saleor/webhook/circuit_breaker/breaker_board.py

```diff
@@ -0,0 +1,221 @@
+import logging
+import time
+from typing import TYPE_CHECKING
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.utils.module_loading import import_string
+
+from saleor.webhook.event_types import WebhookEventSyncType
+
+from ...graphql.app.types import CircuitBreakerState
+
+if TYPE_CHECKING:
+    from ...webhook.models import Webhook
+
+
+logger = logging.getLogger(__name__)
+
+
+# Time frame for webhook events to be analyzed by the breaker board.
+BREAKER_BOARD_TTL_SECONDS: int = 5 * 60
```

Comment:
Is it expected that we will not be able to set these variables without making an explicit change to the code? 
Personally, I would expect, that these values would be configurable from the settings.py - but maybe I missed some reasons why you decided to not have these values as configurable.

Reply:
That was the initial idea, however @patrys insited on hardcoding these settings.
The goal is to make sure that we don't have a dysfunctional defaults and the only valid config is hidden in production env variables.